spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/NacosServiceInstance.java

```diff
@@ -76,6 +76,11 @@ public Map<String, String> getMetadata() {
 		return metadata;
 	}
 
+    @Override
+    public String getScheme() {
+        return getUri().getScheme();
+    }
+
```

Comment:
Please use the correct coding style to pass checkstyle validation
https://sca.aliyun.com/zh-cn/community/developer/contributor-guide/new-contributor-guide_dev#%E9%85%8D%E7%BD%AE-spring-cloud-%E6%A0%87%E5%87%86%E7%9A%84%E4%BB%A3%E7%A0%81%E6%A0%BC%E5%BC%8F

Reply:
And run `mvn clean validate`